game_api.txt

```diff
@@ -338,6 +338,29 @@ To use it, add the `on_screwdriver` function to the node definition.
  * use `on_rotate = screwdriver.disallow` to always disallow rotation
  * use `on_rotate = screwdriver.rotate_simple` to allow only face rotation
 
+
+Sethome API
+-----------
+
+The sethome API adds three global functions to allow mods to read a players home position, set a players home position and teleport a player to home position.
+
+`sethome.get(name)`
+
+* `name` Player who's home position you wish to get
```

Comment:
Please indent these asterisk lines consistently, see screwdriver and stairs API above and below.